provisioning/baremetal_pod.go

```diff
@@ -351,3 +356,118 @@ func createContainerMetal3StaticIpManager(images *Images, config *metal3iov1alph
 	}
 	return container
 }
+
+func newMetal3Volumes() []corev1.Volume {
```

Comment:
Is this list of volumes static? Do we need a function to return it, or could we define it in a variable, too?

Reply:
Done. https://github.com/openshift/cluster-baremetal-operator/pull/45#discussion_r510465229